### PR TITLE
Fix the second launch of Liquibase

### PR DESCRIPTION
**Pull request by nkonev**
## What does this PR do?

Fixes https://github.com/oracle/graalvm-reachability-metadata/issues/244

### CODE REVIEW

**Comment by nkonev** [comment]
cc @dnestoro

File: tests/src/org.liquibase/liquibase-core/4.17.0/src/test/java/org_liquibase/liquibase_core/LiquibaseCoreTest.java [hunk 1]
```diff
@@ -44,6 +44,32 @@ void testYaml() throws Exception {
...
+    @Test
+    void testYamlSecondLaunch() throws Exception {
+        final String jdbcUrl = "jdbc:h2:mem:db1;DB_CLOSE_DELAY=-1";
+        withConnection(jdbcUrl, (connection) -> {
+            Database database = new H2Database();
+            database.setConnection(new JdbcConnection(connection));
+
+            Liquibase liquibase = new Liquibase("changelog.yaml", new ClassLoaderResourceAccessor(), database);
+            liquibase.update();
+            liquibase.update();
```
[the remark this answers is not in the document]
Comment (nkonev) [reply tests/src/org.liquibase/liquibase-core/4.17.0/src/test/java/org_liquibase/liquibase_core/LiquibaseCoreTest.java:31]: Good catch, removed duplication and updated the test

**Comment by nkonev** [comment]
Updated the branch against the latest master, let's merge

**Comment by dnestoro** [comment]
> Updated the branch against the latest master, let's merge

Sorry for a delay, I thought I pressed merge button when I approved PR.

**Pull request merged**